lib/compiler-llvm/src/config.rs

```diff
@@ -1,14 +1,20 @@
 use crate::compiler::LLVMCompiler;
+use anyhow::Result;
```

Comment:
In libraries we should aim to throw specific errors instead of generic ones via `anyhow`, as it makes easier to catch specific errors from the users of the libraries